k8s.yaml

```diff
@@ -158,33 +165,53 @@ spec:
         - name: RAILS_SERVE_STATIC_FILES
           value: "true"
         - name: RAILS_MASTER_KEY
-          value: "7c7b183f59c356cc7277279711d0ace4"
+          valueFrom:
+            secretKeyRef:
+              name: rails
+              key: master-key
         - name: PROJECT_HOSTNAME
           value: "masajid.world"
         - name: POSTGRES_USER
-          value: "masajid"
+          valueFrom:
+            configMapKeyRef:
+              name: postgres-config
+              key: POSTGRES_USER
         - name: POSTGRES_PASSWORD
-          value: "CPnmdlABCmSCl419"
+          valueFrom:
+            secretKeyRef:
+              name: postgres
+              key: password
         - name: POSTGRES_DATABASE
```

Comment:
A further reduction can be achieved if can change the `POSTGRES_DATABASE` env var to `POSTGRES_DB` instead.

Reply:
Do you mean we can we define Postgres env variables in config maps and export all of them for pods that need them?

Reply:
I mean changing `POSTGRES_DATABASE` env var in https://github.com/masajid/masajid/blob/f7c03105641a1b7f099b361a8f913d63e1a69bd5/web_container/config/database.yml#L20 and probably in https://github.com/masajid/masajid/blob/f7c03105641a1b7f099b361a8f913d63e1a69bd5/web_container/.env.example#L4 to be `POSTGRES_DB` so that I can use the existing config map directly since we can't change the env var name in the Postgres container, got it?

Reply:
OK, I see now. Please apply this change in this PR and I will change the application env var and test with the new `k8s.yaml`.

Reply:
Updated!